### PR TITLE
Update German translation

### PR DESCRIPTION
**Pull request by black-sheep-dev**

### CODE REVIEW

File: translations/sailfishos-chum-gui-de.ts [hunk 1]
```diff
@@ -115,25 +115,25 @@
         <location filename="../qml/pages/PackagePage.qml" line="29"/>
         <source>Source code</source>
         <oldsource>Project Repository</oldsource>
-        <translation type="unfinished">Projekt Repository</translation>
+        <translation>Quellcode</translation>
```
Comment (wheelseal) [inline translations/sailfishos-chum-gui-de.ts:118]: Sure? Quellcode is Sourcecode. I think it is better to translate it with 
"Projekt speicherort" or don´t translate it.

Comment (Olf0) [reply translations/sailfishos-chum-gui-de.ts:118]: "Quellcode" is mixed-language, "Quelltext" is the usual, but slightly antiquated German word for "source code".

Hence @black-sheep-dev please use "Quelltext" (or leave it untranslated as "Source code", but not as ~~"Sourecode"~~.

**Review by Olf0 — CHANGES_REQUESTED** [review]
[Line 214](https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/125/files/351abc6a7a2e6cb8b89f761b3db0353a5d6f2731#diff-2d4e79c4fa01a46166921a064442d22dbf755a7cf91d2e1212ecec3816454150R214): Typo, missing capitalisation and "Retrieving" is rather "abrufen" than "empfangen"

My suggestion:  Instead of ~~"Empfange list der verfügbaren Pakete"~~ use
Rufe Liste der verfügbaren Pakete ab

Ditto in lines 226 and 232: ~~"Empfange …"~~ **→** "Rufe  … ab"

**Comment by Olf0** [comment]
Lines 272 and 274:
- Please alter ~~"GUI"~~ to "SailfishOS:Chum GUI App", once in line 272 and twice in line 274: That is how I deliberately phrased the English strings.
- "im SSU" sound strange, closer to the original an better is "für SSU", because it is really the configuration for / of SSU which is altered.

**Comment by Olf0** [comment]
Line 287: You did that well in  line 280, but missed the difference (only "testing"!) here.
**⇒**
Füge SailfishOS:Chum Testing Repository hinzu

**Comment by Olf0** [comment]
Line 299: ~~"Deinstallieren Paket"~~ **→** Deinstalliere Paket

**Review by Olf0 — CHANGES_REQUESTED** [review]
https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/125/files#diff-2d4e79c4fa01a46166921a064442d22dbf755a7cf91d2e1212ecec3816454150R124

File: translations/sailfishos-chum-gui-de.ts (hunk 1 above)
Comment (Olf0) [reply translations/sailfishos-chum-gui-de.ts]: I ultimately settled on "Quelltexte": It seems most natural as quite some Germans tend to say "Source codes" (in plural) when they use the English term.

**Review by Olf0 — APPROVED** [review]
LGTM

**Comment by Olf0** [comment]
@black-sheep-dev, if you want to take the chance to look over these translated strings, again.  I rectified and altered a few in [commit 2fbda80](https://github.com/sailfishos-chum/sailfishos-chum-gui/pull/125/commits/2fbda80252935c835cb841e30727bf8aa839a017), so I would be glad if you review these changes.

**Pull request merged**